src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/TelemetryService.kt

```diff
@@ -19,17 +19,30 @@ class TelemetryService(
   }
 
   fun trackGoalCreateEvent(goal: Goal) {
-    telemetryClient.trackEvent(GOAL_CREATE_EVENT, goal.goalCreateEventCustomDimensions())
-    with(Span.current()) {
```

Comment:
Using the Span API didnt work at all! It didnt sent the 2nd event (but did mess up the custom dimensions in the POST `request` that it was also tracking 😢 )